Android/app/src/main/res/menu/activity_main_drawer.xml

```diff
@@ -2,6 +2,10 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
 
     <group android:checkableBehavior="single">
+        <item
+            android:id="@+id/nav_profile"
+            android:icon="@drawable/icon_profile"
+            android:title="Profile" />
```

Comment:
Don't add constant string here. Move `Profile` to strings.xml & then, use it here.